.dockerignore

```diff
@@ -0,0 +1,2 @@
+/test
+/README
```

Comment:
there are no README folder.
Add another folders to docketignore and add comnets why they should be in dockerignore as it says in requirement "Folders are added to .dockerignore, with explanations. At least 2 big directories should be excluded from build context. Elastic Beanstalk application is initialized."

Reply:
It was kinda template file, forgot to push updated version, sorry. It's done now.